### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by apruneda**
Changes:
- Did an overall edit of the README and CONTRIBUTING pages
- Updated URLs
- Updated branding (Splunk Enterprise SDK)
- Fixed formatting issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,230 +43,171 @@ foreach (var result in searchResults)
 
 ## Supported platforms
 
-.NET 4.5/Mono 3.4, PCL (Windows 8.1, Windows Phone 8.1, iOS (via Xamarin.iOS), Android (via Xamarin.Android)
+This SDK supports .NET 4.5/Mono 3.4, PCL (Windows 8.1, Windows Phone 8.1, iOS (via Xamarin.iOS), and Android (via Xamarin.Android) platforms.
```
Comment (tdhellmann) [inline README.md:46]: Windows Phone is EoS, so maybe we should remove that statement: https://support.microsoft.com/en-us/help/4036480/windows-phone-8-1-end-of-support-faq

**Review by tdhellmann — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,42 +2,34 @@
...
 
-0. Version of this project you're using (ex: 2.1.2)
-1. Platform version (ex: Windows Server 2012)
-1. Framework version (ex: .Net 4.5, Visual Studio 2013)
-1. Splunk version (ex: 6.2.2)
-1. Other relevant information (ex: local/remote environment, Splunk network configuration)
+1. Version of this project you're using (ex: 2.1.2)
+2. Platform version (ex: Windows Server 2012)
+3. Framework version (ex: .Net 4.5, Visual Studio 2013)
+4. Splunk Enterprise version (ex: 8.0)
```
[the remark this answers is not in the document]
Comment (apruneda) [reply CONTRIBUTING.md:14]: done

File: README.md (hunk 1 above)
Comment (apruneda) [reply README.md]: Done

**Comment by EngRajabi** [comment]
please update package in nuget

**Comment by ncanumalla-splunk** [comment]
This SDK is [deprecated](https://github.com/splunk/splunk-sdk-csharp-pcl#deprecation-notice) and no longer under active development.

**Pull request merged**